src/algebra/group/defs.lean

```diff
@@ -602,6 +757,60 @@ instance group.to_cancel_monoid : cancel_monoid G :=
 
 end group
 
+@[ext, to_additive]
+lemma group.ext {G : Type*} ⦃g₁ g₂ : group G⦄ (h_mul : g₁.mul = g₂.mul) : g₁ = g₂ :=
```

Comment:
I guess `group.to_div_monoid` should be injective, but I couldn't manage to prove it.